src/domains/sign/io_sign_user_func.tf

```diff
@@ -0,0 +1,93 @@
+locals {
+  io_sign_user_func = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME                        = "node"
+      WEBSITE_NODE_DEFAULT_VERSION                    = "16.13.0"
+      WEBSITE_VNET_ROUTE_ALL                          = "1"
+      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+      WEBSITE_DNS_SERVER                              = "168.63.129.16"
+      WEBSITE_RUN_FROM_PACKAGE                        = "1"
+      FUNCTIONS_WORKER_PROCESS_COUNT                  = 4
+      AzureWebJobsDisableHomepage                     = "true"
+      NODE_ENV                                        = "production"
+      CosmosDbConnectionString                        = module.cosmosdb_account.connection_strings[0]
+      CosmosDbDatabaseName                            = module.cosmosdb_sql_database_user.name
+      StorageAccountConnectionString                  = module.io_sign_storage.primary_connection_string
+      userUploadedBlobContainerName                   = azurerm_storage_container.uploaded_documents.name
+      userValidatedBlobContainerName                  = azurerm_storage_container.validated_documents.name
+      WaitingMessageQueueName                         = azurerm_storage_queue.waiting_message.name
+      IoServicesApiBasePath                           = "https://api.io.italia.it"
+      IoServicesSubscriptionKey                       = module.key_vault_secrets.values["IOApiSubscriptionKey"].value
+      PdvTokenizerApiBasePath                         = "https://api.uat.tokenizer.pdv.pagopa.it"
+      PdvTokenizerApiKey                              = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
+      NamirialApiBasePath                             = "https://pagopa.demo.bit4id.org"
+      NamirialUsername                                = "api"
+      NamirialPassword                                = module.key_vault_secrets.values["NamirialPassword"].value
+    }
+  }
+}
+
+module "io_sign_user_func" {
+  source = "git::https://github.com/pagopa/azurerm.git//function_app?ref=v3.3.1"
+
+  name                = format("%s-user-func", local.project)
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+
+  # Health check should not be enabled on Premium Functions
+  # https://learn.microsoft.com/en-us/azure/app-service/monitor-instances-health-check?tabs=dotnet#limitations
+  # health_check_path   = "/api/v1/sign/info"
```

Comment:
this is true for elastic premium function apps, for functions app on app service plan we need to enable health checks

Reply:
@pasqualedevita the function needs a key to be invoked. Since there is no way to use a key for healthcheck, the funcion will never be healty.

Is there a workaround for these situations?

Reply:
Yes i n the function definition you need to set authentication anonymous

Reply:
like here https://github.com/pagopa/io-functions-services/blob/master/Info/function.json